src/functions/clientProperties.js

```diff
@@ -11,4 +11,5 @@ module.exports = async client => {
     client.music = new Enmap("music");
     client.bantimers = {};
     client.remindtimers = {};
+    client.gifs = new Enmap("gifs");
```

Comment:
Tbh I think doing this in an enmap is a bad idea. This way the urls are not on the github repo. JSON file might be a better solution.